### ISSUE TITLE
can't run js in my atom

### ISSUE THREAD

**Issue by biolee** [rgbkrk/atom-script#574]
Unable to run

node
Is it in your PATH?

PATH: /usr/bin:/bin:/usr/sbin:/sbin

**Comment by shawninder** [comment]
I get the same since I last updated the plugin (it used to work just fine).

```
$ echo $PATH
/Applications/Sublime Text.app/Contents/SharedSupport/bin/:/Applications/Google Chrome.app/Contents/MacOS/:/Users/shawn/android-sdks/platform-tools/:/Users/shawn/android-sdks/tools/:/Users/shawn/npm/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/opt/X11/bin:/usr/local/git/bin
```

**Comment by rgbkrk** [comment]
How's the 3.x release?

**Comment by shawninder** [comment]
Works for me with the latest update (3.0.1) :+1:

**Comment by grrgrrbla** [comment]
I have pretty much the same issue, and the latest update doesnt help,

I get the following error whenever I try to execute javascript-code:

"Unable to run

node
Did you start Atom from the command line?

  atom .
Is it in your PATH?

PATH: /home/daniel/anaconda3/bin:/home/daniel/anaconda3/bin:/home/daniel/bin:/usr/local/texlive/2014/bin/x86_64-linux:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:usr/bin/atom:usr/bin/nodejs:/usr/bin/atom:/usr/bin/nodejs"

I tried to start atom from the terminal (typing atom .), I have nodejs installed and can use it just fine inside the terminal, I looked at other related issues, nothing worked, I can run python code without any issues. I am using linux mint (3.13.0-37-generic).
It works just fine when I use "configure run options" and type the path of nodejs into the option "Command"

**Comment by shawninder** [comment]
You probably did this, but you need to reload atom for package updates to take effect.

**Comment by grrgrrbla** [comment]
I obviously did "restart" if that is what you mean by reloading?

**Comment by shawninder** [comment]
Yes

**Comment by lincolnfrias** [comment]
In my case (Linux, Ubuntu), the solution was to install node-legacy besides nodejs. I just did:
`sudo apt install  node-legacy`

**Comment by gdad-s-river** [comment]
I'm facing the same problem, I use node via node version manager. Is it that atom is not able to find the node path and hence it is giving the "Unable to run" error.  How can I solve this problem?

**Comment by nixel2007** [comment]
@gdad-s-river you need to have `babel-node` in your path, not `node`

**Comment by stoplion** [comment]
Seems like an important things to mention in the documentation. Also when using NVM setting an alias to babel-node can be tricky..

**Comment by ProVadimir** [comment]
Hello, i have problem with Atom!
help me, please!
![photo](https://user-images.githubusercontent.com/32485847/31126951-0a58a7a4-a856-11e7-9dee-c9cd1ba0ed58.png)

**Comment by nixel2007** [comment]
@ProVadimir do you have babel-node installed?

**Comment by ProVadimir** [comment]
I have another problem!
write me combinations please , so install babel!
![qr3qu_croper_ru](https://user-images.githubusercontent.com/32485847/31128446-844c1dd0-a85a-11e7-980c-8c9bb389c221.png)

**Comment by danielbayley** [comment]
@biolee @shawninder @grrgrrbla @lincolnfrias @gdad-s-river @nixel2007 @stoplion @ProVadimir This has now been addressed in fc84ded7dad08466923071e6d6e4370aa6a36789, and should 'just work' by using the preconfigured Babel setup in this package instead of having to set it up locally every time or worse, globally.

**Comment by rgbkrk** [comment]
Thanks for handling these @danielbayley
